README.md

```diff
@@ -136,14 +136,14 @@ Current release info
 Installing python
 =================
 
-Installing `python` from the `conda-forge/label/python_debug` channel can be achieved by adding `conda-forge/label/python_debug` to your channels with:
+Installing `python` from the `conda-forge` channel can be achieved by adding `conda-forge` to your channels with:
```

Comment:
It seems there is some inconsistency around what channel is used in the README when re-rendering

Can see similar past issues. For example: https://github.com/conda-forge/python-feedstock/commit/9502e319cee429dd67f1027b246a2ae32d1a5517

Think we can ignore this and raise a conda-smithy issue

Reply:
Filed as upstream issue: https://github.com/conda-forge/conda-smithy/issues/2296

Reply:
oh, interesting!

Reply:
If you can post in that issue `conda info`, that would help in debugging

Maybe we can recognize a pattern if a few people share what happened for them and their setup